### PR TITLE
Fix returning values in JSONString scalar

### PR DESCRIPTION
**Pull request by sanket-jethava**
We are not return the graphene.JSONStirng.parse_literal from fields.py of graphql core. Due to this we are unable to take JSONString input of CategoryInput, CollectionInput and CategoryInput. So now we are add return statement and now we will able to run it properly.

Please accept the changes.

Thanks!

I want to merge this change because...

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by korycins — APPROVED** [review]

**Comment by maarcingebala** [comment]
Hey @sanket-jethava, thanks for opening the PR. Can you only keep the changes related to JSONString issue? I would skip the email plugin changes and if we want to add them, open a separate PR with description of these changes.

File: saleor/plugins/user_email/plugin.py [hunk 1]
```diff
@@ -344,14 +344,16 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         configuration = {item["name"]: item["value"] for item in self.configuration}
         self.config = EmailConfig(
-            host=configuration["host"],
-            port=configuration["port"],
-            username=configuration["username"],
-            password=configuration["password"],
+            host=configuration["host"] or settings.EMAIL_HOST,
```
Comment (korycins) [inline saleor/plugins/user_email/plugin.py:347]: I don't think that this is the expected way of handling this. 
This is a plugin that should be configured from the dashboard. Only the Admin plugin needs to have a configuration from settings.

**Review by korycins — CHANGES_REQUESTED** [review]
- I don't think that changes in the email plugins should be the scope of this PR
-

**Comment by sanket-jethava** [comment]
> Hey @sanket-jethava, thanks for opening the PR. Can you only keep the changes related to JSONString issue? I would skip the email plugin changes and if we want to add them, open a separate PR with description of these changes.

Yes, Sure. I will open a seperate PR for the same.

Thanks for verify and quick response.

**Comment by korycins** [comment]
Hey @sanket-jethava you removed a whole file: `saleor/plugins/user_email/plugin.py`. I don't think that this is expected. Let's revert it to a form as it was previously.

**Comment by sanket-jethava** [comment]
> your

@maarcingebala, I think it's done now. please review it.

Thanks!

**Comment by maarcingebala** [comment]
There is still some irrelevant change in this file: 
image
Please revert all changes and keep only the one related to JSONString.

**Comment by sanket-jethava** [comment]
@maarcingebala please verify. I have made changes as you said.

Thanks for quick response and support!

**Review by maarcingebala — APPROVED** [review]

**Review by korycins — APPROVED** [review]

**Comment by sanket-jethava** [comment]
@maarcingebala and @korycins Thank you for approve pull request!

**Comment by maarcingebala** [comment]
Thanks for contributing to Saleor @sanket-jethava!

**Pull request merged**